geometry_msgs/Polygon ros_poly
defecto_zona[] defecto_divpor_zonas

bool reparado
